qcodes/instrument_drivers/tektronix/AWG5014.py

```diff
@@ -1683,11 +1688,10 @@ def _pack_waveform(
             raise TypeError('Marker 2 contains invalid values.' +
                             ' Only 0 and 1 are allowed')
 
-        wflen = len(wf)
-        packed_wf = np.zeros(wflen, dtype=np.uint16)
-        packed_wf += np.uint16(np.round(wf * 8191) + 8191 +
-                               np.round(16384 * m1) +
-                               np.round(32768 * m2))
+        # Note: np.trunc is an order of magnitude faster then np.round
```

Comment:
it seems to me that this note is only relevant in the context of the diff, and if that is true, shall we remove it?

Reply:
I think it's fine to keep. My instinct would be to use round here so an explanation is nice. 

Perhaps we could rephrase it a bit to say. 
```
# Note: we use np.trunc here rather than np.round as it is an order of magnitude faster
```

Reply:
done